CHANGELOG.md

```diff
@@ -32,7 +32,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Vis Builder] Rename wizard to visBuilder in class name, type name and function name ([#2639](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2639))
 - [Vis Builder] Rename wizard on save modal and visualization table ([#2645](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2645))
 - Change save object type, wizard id and name to visBuilder #2673 ([#2673](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2673))
-- Add extension point in saved object management to register namespaces and show filter ([#2656](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2656))
+- [Save Object Aggregation View] Add extension point in saved object management to register namespaces and show filter ([#2656](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2656))
```

Comment:
Does change log need to match with the commit message and the PR title? If so, we don't need to change the existing change log.

![Screen Shot 2022-11-01 at 11 17 32 AM](https://user-images.githubusercontent.com/16925774/199308235-3d2b7528-11d4-49b1-b32c-0a8821eb38c3.png)

Reply:
I added the `[Save Object Aggregation View]` to prefix the name here to indicate the change is associated with this project. Should that be reverted back?

Reply:
@AMoo-Miki @kavilla @joshuarrrr might be able to confirm this as they have more context on `CHANGLOG.md`. If CHANGELOG.md is something similar with Release Notes, then we can't do retrospective changes on existing items(already merged commits).